shared/src/main/java/uk/gov/di/authentication/shared/entity/AuthenticationAttempts.java

```diff
@@ -35,61 +34,42 @@ public AuthenticationAttempts withInternalSubjectId(String internalSubjectId) {
         return this;
     }
 
-    @DynamoDbAttribute("JourneyType")
-    public String getJourneyType() {
-        return journeyType;
-    }
-
     public void setJourneyType(String journeyType) {
         this.journeyType = journeyType;
-        setAuthMethodJourneyType();
+        setJourneyTypeCountType();
     }
 
     public AuthenticationAttempts withJourneyType(String journeyType) {
         setJourneyType(journeyType);
         return this;
     }
 
-    @DynamoDbAttribute("AuthenticationMethod")
-    public String getAuthenticationMethod() {
-        return authenticationMethod;
-    }
-
-    public void setAuthenticationMethod(String authenticationMethod) {
-        this.authenticationMethod = authenticationMethod;
-        setAuthMethodJourneyType();
+    public void setCountType(String countType) {
+        this.countType = countType;
+        setJourneyTypeCountType();
     }
 
-    public AuthenticationAttempts withAuthenticationMethod(String authenticationMethod) {
-        setAuthenticationMethod(authenticationMethod);
+    public AuthenticationAttempts withCountType(String countType) {
+        setCountType(countType);
         return this;
     }
 
     @DynamoDbSortKey
-    @DynamoDbAttribute("AuthMethodJourneyType")
-    public String getAuthMethodJourneyType() {
-        return authMethodJourneyType;
-    }
-
-    public void setAuthMethodJourneyType() {
-        this.authMethodJourneyType = authenticationMethod + "_" + journeyType;
-    }
-
-    public void setAuthMethodJourneyType(String authMethodJourneyType) {
-        this.authMethodJourneyType = authMethodJourneyType;
+    @DynamoDbAttribute("JourneyTypeCountType")
+    public String getJourneyTypeCountType() {
+        return journeyTypeCountType;
     }
 
-    @DynamoDbAttribute("Code")
-    public String getCode() {
-        return code;
+    public void setJourneyTypeCountType() {
+        this.journeyTypeCountType = journeyType + "#" + countType + "#" + "Count";
```

Comment:
Why not do this in the getJourneyTypeCountType() (or getSK() after renaming) method?  You could then remove all these setters and just have a single setter to allow the SK to be set if reading from Dynamo, for example:
setSK(String SK) { this.SK = SK)